lib/transports/ws2.js

```diff
@@ -179,7 +192,12 @@ class WSv2 extends EventEmitter {
     const { sig } = genAuthSig(this._apiSecret, authPayload)
 
     return new Promise((resolve, reject) => {
-      this._ws.once('auth', () => resolve())
+      this.once('auth', () => {
+        debug('authenticated')
+        resolve()
+      })
+
+      debug('authenticating w/ key %s', this._apiKey)
```

Comment:
uh, maybe not good to log access keys security-wise, they can end up in log files etc

Reply:
Removed